kratos/python/add_constraint_to_python.cpp

```diff
@@ -24,36 +24,98 @@ namespace Kratos
 namespace Python
 {
 
-template <class TContainerType, class TVariableType>
-bool HasHelperFunctionConstraint(TContainerType &el, const TVariableType &rVar)
+template <class TVariableType>
+bool HasHelperFunctionMasterSlaveConstraint(MasterSlaveConstraint &el, const TVariableType &rVar)
```

Comment:
I am pretty sure these changes should not be here

Reply:
This is already merged with master, but is comparing with #3453